zstacklib/zstacklib/utils/sanlock.py

```
@@ -219,15 +266,86 @@ def wait(_):
+                self.space_name = value
+            elif key == "resource_name":
+                self.resource_name = value
+            elif key == "timestamp":
+                self.timestamp = int(value)
+            elif key == "checksum":
+                self.checksum = value
+            elif key == "io_timeout":
+                self.io_timeout = int(value)
+
+
+class PaxosLease(object):
```

Pylint (3.3.7)

[refactor] Class 'PaxosLease' inherits from object, can be safely removed from bases in python3

(R0205)